tools/scale-back.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+
+import json
+import traceback
+import numpy
+from PIL import Image, ImageDraw
+import gc
+
+# START OF FIELDS TO BE EDITED
+
+source = 166
```

Comment:
Does it make sense to include image 167? That image is mostly whited out, but there's still a couple of pixels remaining.

Reply:
It is 166 because I wanted shift those from 166 to 164/165 (the current atlas only does until 166.) 

That should be edited by case-by-case basis. If you want to extend 166 to 167 then you change the variables accordingly.